src/components/LinterProvider/index.spec.tsx

```diff
@@ -195,9 +195,11 @@ describe(__filename, () => {
 
   it('renders child content', () => {
     const className = 'childClassName';
-    const root = render({
-      children: jest.fn().mockReturnValue(<div className={className} />),
-    });
+    const children = jest.fn().mockReturnValue(<div className={className} />);
+    const store = configureStore();
+    _loadLinterResult({ store, messages: [] });
```

Comment:
I am wondering if we should mock dispatch here too.. although, that is working as intended. Only thing would be to add a comment here too, probably..